### DISCUSSION TITLE
Error cursor confguration

### DISCUSSION THREAD

**Discussion by peterbuysse** [SafeExamBrowser/seb-win-refactoring#754] [General]
Hi
We orrganised a SEB-examen in our school today. What never happend before: a lot of students weren't able to start SEB-session. I found this message in the logs:

2023-12-11 10:40:47.713 [07] - INFO: Attempting to verify cursor configuration...
2023-12-11 10:40:47.713 [07] - WARNING: Cursor configuration is compromised: 'C:\Users\vonckxs0702\Downloads\fortnite-1\fortnite rocket launcher.cur'! Aborting session initialization...

We found that a program called 'Custom Cursor' was responable: when we removed it, SEB started!

Can we change the settings in the SEB-configuration file to ignore this error?

Thanks!

**Comment by dbuechel** [comment]
Please refer to https://github.com/SafeExamBrowser/seb-win-refactoring/issues/755.

**Comment by tomaastu** [comment]
hola holita

**Reply by tomaastu** [reply]
https://chat.openai.com/

**Reply by tomaastu** [reply]
https://safeexambrowser.org/testing/

**Reply by tomaastu** [reply]
https://www.google.com/

**Reply by tomaastu** [reply]
sebs://safeexambrowser.org/testing/DemoExamMoodleNoPW.seb
